test/integration/puppetlabs/services/jruby/jruby_puppet_internal_test.clj

```diff
@@ -1,60 +0,0 @@
-(ns puppetlabs.services.jruby.jruby-puppet-internal-test
-  (:require [clojure.test :refer :all]
-            [puppetlabs.services.jruby.jruby-puppet-internal :as jruby-internal]
-            [puppetlabs.services.jruby.jruby-testutils :as jruby-testutils]
-            [puppetlabs.services.jruby.jruby-puppet-schemas :as jruby-schemas])
-  (:import (com.puppetlabs.puppetserver.pool JRubyPool)
-           (org.jruby RubyInstanceConfig$CompileMode)
-           (clojure.lang ExceptionInfo)))
-
-(deftest get-compile-mode-test
-  (testing "returns correct compile modes for SupportedJRubyCompileModes enum"
-    (is (= RubyInstanceConfig$CompileMode/JIT
-           (jruby-internal/get-compile-mode :jit)))
-    (is (= RubyInstanceConfig$CompileMode/FORCE
-           (jruby-internal/get-compile-mode :force)))
-    (is (= RubyInstanceConfig$CompileMode/OFF
-           (jruby-internal/get-compile-mode :off))))
-  (testing "returns a valid CompileMode for all values of enum"
-    (doseq [mode jruby-schemas/supported-jruby-compile-modes]
-      (is (instance? RubyInstanceConfig$CompileMode
-                     (jruby-internal/get-compile-mode mode)))))
-  (testing "throws an exception if mode is nil"
-    (is (thrown? ExceptionInfo
-                 (jruby-internal/get-compile-mode nil))))
-  (testing "throws an exception for values not in enum"
-    (is (thrown? ExceptionInfo
-                 (jruby-internal/get-compile-mode :foo)))))
-
-(deftest ^:integration settings-plumbed-into-jruby-container
```

Comment:
I think @rlinehan and I decided it was kind of redundant, since tests like `general-puppet-int-test` tickle the code paths that would rely on those settings having been plumbed through (right?), but I think it'd be reasonable to add an integration test that tests this specifically